vividus-plugin-mobile-app/src/main/java/org/vividus/bdd/mobileapp/steps/TouchSteps.java

```diff
@@ -142,6 +160,36 @@ public void swipeToElement(SwipeDirection direction, Locator locator, Duration d
                 ComparisonRule.EQUAL_TO, 1);
     }
 
+    /**
+     * Select a next or previous picker wheel value in date picker
+     * @param direction direction of next value, either <b>NEXT</b> or <b>PREVIOUS</b>
+     * @param locator locator to find a <i>XCUIElementTypePickerWheel</i> element
+     */
+    @When("I select $direction value in picker wheel located `$locator`")
```

Comment:
it doesn't look like `touch` step (no gestures), what about `ElementSteps`?

Reply:
from end-user pov and from underlying mechanics we tap on picker wheel to choose particular value